### PR TITLE
Filter fix and search edgecase

### LINKED ISSUE #134: Filter by Type not working
## Expected Behavior
Filter pokemon by type

## Actual/Current Behavior
Pokemon does not show up after filtering

## Steps to Reproduce the Problem
  1. Open app
  2. Filter by pokemon type
  3. Observe

## Pre-Testing TODOs
  1. Check if PokeList had FilterBar imported correctly.

## Testing Steps
  1. Navigate to home
  2. Click on filter by type
  3. Observe

### PR DESCRIPTION
**Pull request by connielion**
## Changes
1. Fixed filtering by renaming incorrectly named functions.
2. Added `trim()` to the search bar value.

## Purpose
Fix filtering bug and added trim to take care of whitespaces in user input.

## Approach
Filtering function stopped working when Filter component was renamed FilterBar with "find all and replace all" function. Fixed by renaming the methods that are called when user filters.

Closes #134

### CODE REVIEW

**Review by BlackCubes — APPROVED** [review]
Nice catch on the bug!

**Review by Lap343 — APPROVED** [review]
Good job

File: src/components/PokeApp.js [hunk 1]
```diff
@@ -18,7 +18,7 @@ const PokeApp = () => {
   const [isLoading, setIsLoading] = useState(true);
   // search bar onChange function for searching pokemon & conditional rendering
   const handleNameSearch = (e) => {
-    const searchBarValue = e.target.value;
+    const searchBarValue = e.target.value.trim();
```
Comment (krferrell) [inline src/components/PokeApp.js:21]: Never used the trim method! That's cool. Thanks for teaching me.

**Review by krferrell — APPROVED** [review]
Nice catch!

**Pull request merged**